### ISSUE TITLE
Color docs: Update tab names for consistency

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon-website#3528]
- [x] Rename Usage to Tokens
- [x] Rename Implementation to Usage
- [x] Tab order for Color should be Overview, Usage, Tokens, Code
- [x] Add redirect for Implementation->Usage

<img width="822" alt="Screenshot 2023-05-03 at 9 12 03 AM" src="https://user-images.githubusercontent.com/2753488/235942076-92294c09-599b-4b5d-b08d-03e50808f2fa.png">

https://www.figma.com/file/mtARocDBevZmtgEMjb1cJs/2x-grid-overview-updates?node-id=0-1&t=EjP5Om4KoQC3al2O-0
